### DISCUSSION TITLE
Firefox or chromium

### DISCUSSION THREAD

**Discussion by anshiq** [termux/termux-packages#10341] [Q&A] [answered]
Please  tell me  anybody working on firefox or chromium.

**Comment by macerg** [comment]
Or install proot-distro and then install your preferred flavour of Linux.
I have Debian bullseye installed. It's less than 2GB with Firefox installed.

**Comment by J0K3RS-L4UGH** [comment]
I had both installed, but deleted Firefox, so in my opinion, chrome

**Reply by anshiq** [reply]
@Gomenburu  we are not talking about  installation through proot distro !!!!!!!!!

**Comment by ghost** [comment, marked as answer]
We currently don't have plans to package both of these browsers yet due to the complexity of the codebase. especially if the source code is more than 100GB (regardless on the archive size. it would fetch dependencies on the internet if building). CI would eventually run of of space after during building. so someone will have to open a PR. it would be better off running these apps in prooted distros instead

**Reply by anshiq** [reply]
can you work on Blue Fish editor, I have opened an issue. All details are given there. It is great code editor.
